packages/coinstac-ui/app/render/state/ducks/maps.js

```diff
@@ -30,15 +34,33 @@ const INITIAL_STATE = {
 //   {type: 'boolean', name: 'isControl'},
 //   {type: 'number', name: 'age'}
 // ]
```

Comment:
can we get rid of the comments above? I know it wasnt this pr but I dont think they need to be here

Reply:
Removed.